lightify/__init__.py

```diff
@@ -917,8 +917,12 @@ def __init__(self, host, new_device_types=None):
         self.__device_types.update(new_device_types or {})
 
         self.__logger = logging.getLogger(MODULE)
-        self.__logger.addHandler(logging.NullHandler())
-        self.__logger.info('Logging %s', MODULE)
+        self.__logger.setLevel(logging.INFO)
+        if loghandler is not None:
+            self.__logger.addHandler(loghandler)
+        else:
+            self.__logger.addHandler(logging.NullHandler())
+        self.__logger.info("Initializing python %s, version=%s", MODULE, __version__)
```

Comment:
Just to make it 100% consistent with the rest of code:

```suggestion
        self.__logger.info('Initializing %s, version=%s', MODULE, __version__)
```